aiidalab_widgets_base/viewers.py

```diff
@@ -331,9 +337,10 @@ def change_camera(change):
             [supercell_selector, background_color, camera_type, center_button]
         )
 
-    @observe("cell")
-    def _observe_cell(self, _=None):
-        if self.cell:
+    @observe("ase_structure")
+    def _observe_ase_structure(self, _=None):
+        if self.ase_structure:
```

Comment:
@unkcpz  In this logic, please take into account non-periodic systems by checking the pbc attribute of the ase object.

Reply:
Good point! @danielhollas 

I add a check and only parse and attach symmetry when pbc shows it is 3-D

Reply:
@unkcpz it might also be nice to automatically hide the Cell tab in that case. Not sure how much work that would be so perhaps in another PR?

Reply:
Probably can update the tab widget but then need to bring it back when 3D structure is selected again, seems a bit tedious to make it work correctly, I'll take a look. 
Or we can make the widget support  the 2D and 1D with big cell set like we have for [optimade-client]?(https://optimadeclient.materialscloud.io/). From where if you search structure in MC2D database. Not sure I remember how we do that, I'll check it.

Reply:
```suggestion
        if self.ase_structure and any(self.ase_structure.pbc):
```